flocker/node/agents/test/test_blockdevice.py

```diff
@@ -633,6 +640,48 @@ def test_ignore_deleted_datasets(self):
         self.assertEqual(expected_changes, actual_changes)
 
 
+class BlockDeviceDeployerResizeCalculateNecessaryStateChangesTests(
+        SynchronousTestCase
+):
+    """
+    Tests for ``BlockDeviceDeployer.calculate_necessary_state_changes``
+    in the cases relating to resizing a dataset.
+    """
+    def test_maximum_size_unchanged(self):
+        """
+        ``BlockDeviceDeployer.calculate_necessary_state_changes`` does not
+        calculate a change to shrink or grow datasets if the maximum_size of
+        the dataset has not changed.
+        """
```

Comment:
This test will probably not be able to pass in this branch.  I deleted some similar tests from the FLOC-1582 branch in afa1ffe1bf3c1a24bbd4457ac1b69dba8abe195a.